lib/plugins/main.js

```diff
@@ -190,30 +190,6 @@ export default async ({ res }, inject) => {
 
   <% } %>
 
-  /** --------------------------------------------------------------------------------------------- **/
```

Comment:
All this needs to stay here in this file since it creates the service worker in the first place.